jdi-light-vuetify-tests/src/main/java/io/github/com/custom/CompositeLabelChip.java

```diff
@@ -0,0 +1,40 @@
+package io.github.com.custom;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import io.github.com.custom.asserts.CompositeLabelChipAssert;
+
+import java.util.List;
+
+public class CompositeLabelChip extends UIBaseElement<CompositeLabelChipAssert> {
```

Comment:
CompositeLabelChip should be extension for Chip (we use common methods + add methods for customization)
For example, now we can't use methods like selected()/deselected() and others